internal/covid/covidclient/client.go

```diff
@@ -0,0 +1,61 @@
+package covidclient
```

Comment:
ทำเป็น mockgen ตัว mock น่าจะง่ายกว่า

```suggestion
//go:generate mockgen -source=./client.go -destination=./mock/mock_client.go -package=mock

package covidclient
```

Reply:
แบบนี้ไม่เคยใช้เลย เดี๋ยวลองดูครัช